CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## Version 1.0.2 (2020-01-10)
+- Speed up `Tree.splits` ([#349][349], [@treeowl][treeowl])
+- Speed up `Gen.shuffle` ([#348][348], [@treeowl][treeowl])
+- Add docs on the bounds of `Size` ([#346][346], [@chris-martin][chris-martin])
+- Fix performance issues with color handling ([#345][345], [@stolyaroleh][stolyaroleh])
+- Add `mapMaybe`, `mapMaybeT`, in `Tree` and `Gen` ([#339][339], [@treeowl][treeowl])
+- Fix some formatting bugs in Haddock ([#332][332], [@sshine][sshine])
+- Add `MonadGen` instances for `StateT` ([#330][330], [@symbiont-sam-halliday][symbiont-sam-halliday])
```

Comment:
I contributed to this through the #321

Reply:
Yeah and that is probably the most important/impressive change of this release actually!

I think we should add https://github.com/hedgehogqa/haskell-hedgehog/pull/321 and also add @HuwCampbell to the above line.

Reply:
Probably add tomjaguarpaw as well, most of it was from his investigations./

Reply:
Oh, I think the reason I didn't add #321 was because it was shown under the previous release PR. I'll for sure add you there as well as tomjaguarpaw.

Reply:
Done :confetti_ball: